### PR TITLE
Update various typos, phrasing & punctuation.

### PR DESCRIPTION
**Pull request by preserveddarnell**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -12,7 +12,7 @@ furnished to do so, subject to the following conditions:
 The above copyright notice and this permission notice shall be included in all
 copies or substantial portions of the Software.
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+THE SOFTWARE IS PROVIDED "AS-IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
```
Comment (seanyesmunt) [inline LICENSE]: We shouldn't change this. It is the official MIT license.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 The LBRY app is a graphical browser for the decentralized content marketplace provided by the
 [LBRY](https://lbry.io) protocol. It is essentially the
-[lbry daemon](https://github.com/lbryio/lbry) bundled with an UI using
+[lbry daemon](https://github.com/lbryio/lbry) bundled with a GUI using
```
Comment (seanyesmunt) [inline README.md]: I think we should keep `UI`. But maybe change to `bundled with a UI`

**Comment by seanyesmunt** [comment]
Can we [send you some LBC as appreciation](https://lbry.io/faq/appreciation)?

File: LICENSE (hunk 1 above)
Comment (preserveddarnell) [reply LICENSE]: Ah ok, I thought it was non-immutable.

**Pull request merged**